src/main/steps/tasklistresponse/miam/MIAMMapper.ts

```diff
@@ -0,0 +1,43 @@
+import { CaseWithId } from '../../../app/case/case';
+import { Miam, Respondent, YesOrNo } from '../../../app/case/definition';
+import type { AppRequest } from '../../../app/controller/AppRequest';
+
+export const setMIAMDetails = (respondent: Respondent, req: AppRequest): Respondent => {
+  let miamFromResponsent: Miam;
+  if (respondent?.value?.response && respondent?.value?.response?.miam) {
```

Comment:
Do we need both condition checks? or only one 'respondent?.value?.response?.miam' can suffice?

Reply:
will remove this